stable/admin/templates/statefulset.yaml

```diff
@@ -27,8 +27,15 @@ spec:
   template:
     metadata:
       name: {{ template "admin.fullname" . }}
-      {{- if .Values.admin.podAnnotations }}
       annotations:
+      {{- if .Values.admin.configFiles }}
+        {{- with .Values.admin.configFiles }}
+        checksum/config: {{ toYaml . | sha256sum }}
```

Comment:
Is the issue here is that changes to volume mounts are not detected?

Reply:
Yes, I think that this is a known issue - https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments